packages/xstate-wallet/src/workflows/approve-budget-and-fund.ts

```diff
@@ -228,6 +228,7 @@ const createLedger = (store: Store) => async (context: Initial): Promise<LedgerI
   const ledgerId = entry.channelId;
   await store.setFunding(entry.channelId, {type: 'Direct'});
   await store.setLedger(entry.channelId);
+  await store.setApplicationSite(ledgerId, context.budget.domain);
```

Comment:
You will need to remove the `setApplicationSite` call from the `application` workflow.

Reply:
Isn't `SetApplicationSite` setting the site for the application channel in the application workflow?

Reply:
oh, right